### ISSUE TITLE
Mosviz new "ghost": select one object, change layer properties, select a different object, flashing gets triggered

### ISSUE THREAD

**Issue by camipacifici** [spacetelescope/jdaviz#446]
Testing the NIRISS parsers, I found a ghost. It gets triggered after changing the layer properties on one object and then switching to a different object.
It is shown in this video: https://stsci.box.com/s/l1csxxdrvqj9f71ysfcsy0jb2yzxgnxs

@astrofrog, @mariobuikhuizen, @maartenbreddels, maybe have ideas?

**Comment by mariobuikhuizen** [comment]
At first I couldn't reproduce this issue, but I did see an error. While investigating the error I arrived at the same spot that was problematic with the update loop fixed in 95f3e8e201ee105cf436a3a1f69543b6e2003d41. After checking out the commit before this merged Pr (#427), I could reproduce this issue, so updating to main will solve this.

I will look further into the error that is still being produced.

Minimal steps to reproduce the error:

- Click on a row in the table
- Click in the cog in the toolbar of top right viewer
- Click on another row in the table
